Sources/NestKit/NestInfoController.swift

```diff
@@ -19,6 +19,24 @@ public struct NestInfoController {
         return nestInfo
     }
 
+    /// Get NestInfo.Command from `owner/repository`
+    public func fetchCommand(reference: String, version: String) -> NestInfo.Command? {
```

Comment:
Ditto (naming)

Reply:
Thanks!
I've made the corrections here. (https://github.com/mtj0928/nest/pull/36/commits/cbbe01596f2d30b35e16baa133f80bdf86343881)